tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -441,6 +509,20 @@ def test_multiple_nfs_based_PVs(
         11:- Deletion of Pods and PVCs
 
         """
+        if not self.nfs_client_ip:
+            pytest.skip(
+                "Skipped the test as a valid nfs client ip is required, "
+                " for nfs outcluster export validation. "
```

Comment:
This is also repeated 4times, this can be defined in method and you can just call this method at the start of each test which require the connection to NFS client.

Reply:
Addressed in latest commit.